### ISSUE TITLE
How do I get others to connect to the server?

### ISSUE THREAD

**Issue by Carnagion** [smithy212000/PortMiner#24]
So I did everything that the Usage section tells me to. PortMiner and my Minecraft server runs just fine.
I found that I am able to connect to my server by typing in either "**:25565**" or "**[my IPv4 address]:25565**" or "**localhost**" (makes sense, since it's my computer I'm hosting it on).
However, neither the wiki nor the Planet Minecraft page tell me how others will connect to the server. 
In other words, I'm asking what other people will need to type into the server address box in Minecraft so that they can connect to the server.

It'd be great if you can reply soon, thanks a lot.

**Comment by ClockTheSenior** [comment]
I also have the same issue rn!
Does PortMiner map 25565 to some other port maybe? If yes, then which port??

**Comment by Atitla** [comment]
> I also have the same issue rn!

Have you found anything since then pls ?

**Comment by ClockTheSenior** [comment]
nope
